### ISSUE TITLE
Not working with quickfix lists

### ISSUE THREAD

**Issue by edwardbaeg** [SUSTech-data/wildfire.nvim#7]
The default mapping `<CR>` overrides the default behavior of the quickfix. Instead of opening the file, it throws this error:
```
E5108: Error executing lua: ...r/neovim/0.9.1/share/nvim/runtime/lua/vim/treesitter.lua:190: attempt to index local 'node_or_range' (a nil value)
stack traceback:
        ...r/neovim/0.9.1/share/nvim/runtime/lua/vim/treesitter.lua:190: in function 'get_node_range'
        ...cal/share/nvim/lazy/wildfire.nvim/lua/wildfire/utils.lua:13: in function 'get_range'
        ...ocal/share/nvim/lazy/wildfire.nvim/lua/wildfire/init.lua:30: in function 'unsurround_coordinates'
        ...ocal/share/nvim/lazy/wildfire.nvim/lua/wildfire/init.lua:78: in function 'update_selection_by_node'
        ...ocal/share/nvim/lazy/wildfire.nvim/lua/wildfire/init.lua:96: in function 'init_by_node'
        ...ocal/share/nvim/lazy/wildfire.nvim/lua/wildfire/init.lua:104: in function <...ocal/share/nvim/lazy/wildfire.nvim/lua/wildfire/init.lua:101>
```

**Comment by fecet** [comment]
Could you provide some detail about how to reproduce？This is expected to be solved at #4

**Comment by edwardbaeg** [comment]
Sorry, I just now noticed that my plugin manager had issues updating `wildfire.nvim` and therefore I was on a stale version without the linked fix. I have updated and it's working as expected! Thank you!
